CHANGELOG.md

```diff
@@ -6,7 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [2.3.0] - 2017-11-14
+
 ### Added
+- Switch from local token info service to remote one
```

Comment:
@adyach I think it's the other way around "switch from remote token info to local token info service"

Reply:
but the default one is local, so you can switch to remote by turning the feature on

Reply:
ok, got it